cfme/tests/infrastructure/test_vm_migrate.py

```diff
@@ -74,7 +76,33 @@ def test_vm_migrate_should_create_notifications_when_migrations_fail():
         initialEstimate: 1/4h
         startsin: 5.10
         tags: service
+        testSteps:
```

Comment:
This is a negative test case right? Can you add that in the Polarion `metadata`? 

`caseposneg: negative`